src/django_peerctl/management/commands/peerctl_set_peer_session_type.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+from fullctl.django.management.commands.base import CommandInterface
+
+from django_peerctl.models.peerctl import PeerSession, Port
+
+
+class Command(CommandInterface):
+    """
+    Sets the peer_session_type of a PeerSession based on the ports properties
+    """
+
+    def run(self, *args, **kwargs):
+        qset = PeerSession.objects.all()
+
+        for session in qset:
+            port = Port().first(id=session.port.id)
+
+            pni_regex = re.compile(r"\bPNI\b", re.IGNORECASE)
+            if port.is_ixi:
+                session.peer_session_type = "ixp"
+            # if "PNI" or "pni" in port.virtual_port_description:
+            elif port.virtual_port_description and pni_regex.search(
+                port.virtual_port_description
+            ):
+                session.peer_session_type = "pni"
+            else:
+                session.peer_session_type = "transit"
+
```

Comment:
@jason20c lets also track and output when a session is changed

so self.stdout.write some info message if a session.peer_session_type is changed by the command

Reply:
The info message should contain session, port ip4, port ip6, original value and new value